certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/action/certificate/model/CertificateActionFactory.java

```diff
@@ -127,7 +130,9 @@ public static CertificateAction create(CertificateActionSpecification actionSpec
                   new ActionRuleChildRelationNoMatch(List.of(RelationType.REPLACE),
                       List.of(Status.REVOKED)),
                   new ActionRuleUserNotBlocked(),
-                  new ActionRuleNoComplementMessages()
+                  new ActionRuleNoComplementMessages(),
+                  new ActionRuleProtectedPerson(),
+                  new ActionRuleInactiveUnit()
```

Comment:
new ActionRulePatientAlive() should also be considered when replacing. 

Suggest we don't include it in REPLACE_CONTINUE as the draft has already been created and there is no rule for not allowing the user to sign a draft even if the patient is deceased.